include/linux/double-fetch-detection.h

```diff
@@ -0,0 +1,17 @@
+#ifndef __LINUX_DOUBLE_FETCH_DETECTION_H__
+#define __LINUX_DOUBLE_FETCH_DETECTION_H__
+#ifdef CONFIG_DF_DETECTION
+
+struct address{
+    void *startaddress;
+    unsigned long len;
+};
+
+
+void addAddress(void* addr,unsigned long len);
```

Comment:
I suggest you take a look at https://www.kernel.org/doc/html/v4.10/process/coding-style.html
Your style is ok for any other language (or even another C project), but it is inconsistent with the rest of the kernel.
In particular, look at the "Naming" section.
E.g. this function should be called `add_address`, the next one - start_system_call, etc.

It's important that we fix this all now, before you write too much code, because the kernel maintainers will make us change the style anyway :)

Reply:
Okay i will fix the naming and follow the naming conventions.
Thanks